Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -148,6 +154,26 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
     }()
 
     var pocketStories: [PocketStory] = []
+    
+    var hasRecentBookmarks = false
+    var isRecentlySavedSectionEnabled: Bool {
+        get {
+            profile.places.getRecentBookmarks(limit: 5).uponQueue(.main) { [weak self] result in
+                if let bookmarks = result.successValue,
+                   !bookmarks.isEmpty,
+                   !BookmarksHelper.filterOldBookmarks(bookmarks: bookmarks, since: 10).isEmpty {
+                    self?.hasRecentBookmarks = true
+                } else {
+                    self?.hasRecentBookmarks = false
+                }
+            }
+            
+            return AppConstants.IS_RECENTLY_SAVED_SECTION_ENABLED
```

Comment:
nit: You could do a guard on `AppConstants.IS_RECENTLY_SAVED_SECTION_ENABLED` so you don't do any places queries if the feature isn't enabled